pages/virtual-leagues/[type].tsx

```diff
@@ -1,32 +1,45 @@
 import { NextPage, NextPageContext } from "next";
 import VirtualLeaguePageContent from "../../components/virtual-leagues/VirtualLeaguePageContent";
-import {
-  getVerifiedEntries as getVirtualLeagues,
-  GetVerifiedEntriesResponse,
-} from "../../services/verified";
+import { getVerifiedEntries, GetVerifiedEntriesResponse, } from "../../services/verified";
 import { toVerifiedType, VerifiedType } from "../../services/VerifiedType";
 import { getVerifiedExtraInformation } from "../../utils/verifiedTypeHelper";
 
-interface PageProps {
+type LegalVerifiedType = {
```

Comment:
pirk: interface

Reply:
Can do!

(for egen læring: hva er gevinsten i dette scenariet?)

Reply:
husker ikke helt tbh, men sist gang jeg leste meg opp på det sa internett at du burde alltid bruke interface der du kan, og bruke type når du må (feks når du lager union). Også litt fordi det er det som gjøres i resten av repo da

Reply:
Ja, ok.  Enig i at abstraksjoner gir mening hvis man ser for seg å ha mange forskjellige implementasjoner. Lettere å plugge på andre ting uten å måtte gjenbruke typen. Kan begynne å bruke interfacer over typer der jeg kan! Takk!